examples/EMTemplates/EM_GeneralPerfExperiments_Template.json

```diff
@@ -0,0 +1,206 @@
+[
+  {
+    "experiment_name": "quarkus-resteasy-autotune-min-http-response-time-db",
+    "resource": {
```

Comment:
can we add 
"containerName" and 
"imageName" here as it remain constant across all trials .